locodedb/calls.go

```diff
@@ -0,0 +1,94 @@
+package locodedb
+
+import (
+	"errors"
+	"fmt"
+	"strings"
+)
+
+var errRecordNotFound = errors.New("record not found")
+
+type LOCODE [2]string
+
+var (
+	mCountries map[CountryCode]string
+
+	mLocodes map[string]locodesCSV
+)
+
+// Get returns a record for a given LOCODE string.
+func Get(locodeStr string) (rec Record, err error) {
```

Comment:
named returned values not used, lets drop them. i would say using them is a bad practice